src/server/index.js

```diff
@@ -18,7 +18,7 @@ import { createApiServer, isActionFromAuthenticPlayer } from './api';
 const PING_TIMEOUT = 20 * 1e3;
 const PING_INTERVAL = 10 * 1e3;
 
-export function Server({ games, db, _clientInfo, _roomInfo }) {
+export function Server({ games, db, _clientInfo, _roomInfo, log }) {
```

Comment:
Instead of parameterizing `Server` (which will get messy now that you have to pass this down many layers after #251), how about we just expose a logging module that is imported here and used? I don't actually imagine that people will want anything other than `console.log`, so instead of allowing a custom logger implementation, let's just turn it off in production builds?

Also, I think different logging levels make sense, which is easy to do if we just add two functions in the logging module:

```
// turned off in prod builds
// used for general diagnostic messages
logging.info()

// always on
// signals an unexpected value or other error
logging.error()
```